bootstrap/main.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import pathlib
+import shutil
+import sys
+import time
+from typing import Any
+from warnings import warn
+
+import docker
+
+client = docker.from_env()
+low_level_api = docker.APIClient(base_url="unix://var/run/docker.sock")
+
+DEFAULT_FILE_PATH = pathlib.Path("/bootstrap/startup.json.default").as_posix()
```

Comment:
I'm not sure if I follow, but shouldn't this be absolute ? `absolute()`

Reply:
this is already absolute

Reply:
so, why posix ? If you want a string you could use `str()` or `absolute()`, the posix call may get the attention of someone that is reading the code asking why did you use the posix function for that ? And that's my initial question while reading the code.
both str and absolute are a clear way to express what you want, or is there a really deep meaning behind the `as_posix` call that I'm not aware in your implementation ?

Reply:
absolute() is apparently undocumented/unsupported. str() works fine, I'll switch to that.